requirements.in

```diff
@@ -2,10 +2,11 @@ attrs==22.1.0
 authlib==1.1.0
 cattrs==22.2.0
 click==8.1.3
+flake8<5 # pytest-flake8 does not support flake8 5+
```

Comment:
pytest-flake8 is abandoned for almost a year now, community consensus seems to be that it's better to use flake8 directly if we can. I'll handle that in a separate PR.